src/services/i18n/I18n.ts

```diff
@@ -0,0 +1,40 @@
+import { createI18n } from '@kong-ui-public/i18n'
+
+import { camelCaseToWords } from '@/utilities/camelCaseToWords'
+import { get } from '@/utilities/get'
+
+interface I18nRecord {
+  [key: string]: I18nRecord | string
+}
+
+class I18nError extends Error {
+  get key() {
+    return this.message
+  }
+}
+
+export default <T extends I18nRecord>(strs: T) => {
```

Comment:
**Suggestion**: Could we call this `messages` like the parameter is called in `@kong-ui-public/i18n`?

```suggestion
export default <T extends I18nRecord>(messages: T) => {
```